server/models/Dog.js

```diff
@@ -69,4 +73,5 @@ const dogSchema = new Schema({
 
 const Dog = model("Dog", dogSchema)
 
-export default Dog
\ No newline at end of file
+// export default Dog
```

Comment:
```suggestion
export default Dog
```

Reply:
Since we're using ES6, we have to use `export` statements, `module.exports` won't work.